MarkBind/markbind
# Add navigation documentation for previous/next page navigation

**What is the purpose of this pull request?**


**Overview of changes:**
Fixes #1975

**Anything you'd like to highlight/discuss:**
I implemented the 1st solution suggested by @tlylt in the #1975 issue where I documented a common way to add page navigation to a project quickly. Documentation is broken down into 2 parts namely, Adding the required file and folder and Editing the content files, where code examples have been added for clarity.

**Testing instructions:**
Navigate to the `Adding Navigation Buttons` subsection under the `Working with sites` section to verify the documentation: https://deploy-preview-2048--markbind-master.netlify.app/userguide/addingnavigationbuttons

**Proposed commit message: (wrap lines at 72 characters)**
Add documentation for previous/next page navigation

---

**Checklist:** :ballot_box_with_check:
